python/usb_console/run_simulation.py

```diff
@@ -210,6 +208,8 @@ def stop_all(self, reason_for_stop):
             radios_config = config_data["radios"]
         with open(os.path.join(os.path.dirname(os.path.abspath(__file__)), 'configs/radio_keys.json')) as radio_keys_config_file:
             radio_keys_config = json.load(radio_keys_config_file)
+        with open(os.path.join(os.path.dirname(os.path.abspath(__file__)), 'configs/flask_keys.json')) as flask_keys_config_file:
+            flask_keys_config = json.load(flask_keys_config_file)
```

Comment:
It would be nice if these config paths were read in as command-line arguments, like the base config file is read. Not necessary though.